pages/callback.tsx

```diff
@@ -107,7 +107,7 @@ const CallbackPage: React.FC = () => {
         return setError(api_key.error);
       }
 
-      window.location.replace(authResult.appState ?? '/'); // router.replace changes the "slug" query parameter so we can't use it here, because this parameter is used when calling the /api/preview endpoint
+      window.location.replace(getCookie(urlAfterAuthCookieName) ?? '/'); // router.replace changes the "slug" query parameter so we can't use it here, because this parameter is used when calling the /api/preview endpoint
```

Comment:
This should already be in the `main` branch. Just rebase on it.

Reply:
rebased